core/protos/o2control.proto

```diff
@@ -375,17 +375,17 @@ message GetWorkflowTemplatesRequest{
 }
 message VarSpecMessage {
     enum UiWidget {
-        editBox = 0;
-        slider = 1;
-        listBox = 2;
+        editBox = 0;    // plain string input line, can accept types number and string
+        slider = 1;     // input widget exclusively for numbers, range allowedValues[0]-[1]
+        listBox = 2;    // displays a list of items, can accept types number, string or list; if number/string ==> single selection, otherwise multiple selection allowed
         dropDownBox = 3;
         comboBox = 4;
     }
 
     enum Type {
-        bool = 0;
```

Comment:
These changes need to be followed by an update in the enum converter in `serverutils.go`. @graduta do we keep the same defaults as before? i.e. `UiWidget` defaults to `editBox` and `Type` defaults to `string`?

Reply:
Yes, we just discussed this. I reordered the enum to make `string` the first entry so that `string` becomes the implicit Protobuf default regardless of what the code does.

I will amend `serverutils.go`.

Reply:
ok pushed the change